src/commands/status.js

```diff
@@ -34,15 +34,19 @@ const STATUSES = {
 class StatusCommand extends Command {
   async run() {
     const {flags} = this.parse(StatusCommand);
-    const {repo, owner, token, url, deployment, ...rest} = flags;
+    const {repo, owner, token, env, url, deployment, 'auto-inactive': autoInactive, ...rest} = flags;
 
     const options = {
       ...rest,
+      environment: env || rest.environment,
       mediaType: {previews: [PREVIEWS.ANT_MAN, PREVIEWS.FLASH]},
     };
 
+    options.auto_inactive = autoInactive;
+
     if (url) {
       options.log_url = url;
+      options.environment_url = url;
```

Comment:
I've bundled these two together currently, partly because I can't currently see the use case for logging two different URLs, but I'm happy to be corrected. On further investigation I have concluded that `log_url` will populate the "Deployed by " message on the environments screen, while `environment_url` controls the "View Deployment" button on the right of each deployment

Reply:
love it 🔥 sorry I was on vacation  🤓 I'll take a look at this today